app/src/app/home/pages/suite/suite.model.ts

```diff
@@ -35,7 +35,7 @@ export class SuitePageItem extends PageListItem<
   public eventDate(): Date {
     switch (this.type) {
       case SuitePageItemType.Batch:
-        return this.asBatch().submittedAt;
+        return this.asBatch().submittedAt as unknown as Date;
```

Comment:
Could you please elaborate why this pattern is needed? I see that in a few other files as well.

Reply:
Marking this comment as resolved, in favor of your earlier explanation. We can track this discussion there.